daemon-tests/tests/none-collaborative_settlement.rs

```diff
@@ -0,0 +1,55 @@
+use daemon::projection::CfdState;
```

Comment:
nit: `non-collaborative-settlement` would be a much better name :P (I would also remove mix of hyphens and underscores, but I understand if you feel different).

Reply:
I think files (modules) have to respect the [naming rules](https://rust-lang.github.io/api-guidelines/naming.html), so snake_case it is.